rotkehlchen/tests/api/test_accounting_rules.py

```diff
@@ -218,9 +218,9 @@ def test_manage_rules(rotkehlchen_api_server, db_settings):
     ]
 
     # update rule 2 to a combination that already exists
-    rule_2['event_type'] = rule_1['event_type']
-    rule_2['event_subtype'] = rule_1['event_subtype']
-    rule_2['counterparty'] = rule_1['counterparty']
+    rule_2['event_type'] = rule_1['event_type']  # type: ignore
```

Comment:
why are you using type ignore around here and above? This should only be used in the absence of any other solution